### PR TITLE
IBX-380: Cannot deselect the subtree in Admin -> Roles

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-380
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/themes/admin/user/role_assignment/create.html.twig [hunk 1]
```diff
@@ -112,7 +111,12 @@
                         {{ form_widget(form.limitation_type.location, {'label': 'limitation_type.subtree'|trans|desc('Subtree'), 'attr': {'class': 'ez-limitations__radio ez-input ez-input--radio', 'data-disable-selector': '#role_assignment_create_locations_select_content'}, 'label_attr': {'class': 'ez-limitations__label radio-inline'}}) }}
                         {{ form_widget(form.locations) }}
                         <div class="ez-limitations__item-subtree">
-                            <button id="role_assignment_create_locations_select_content" data-udw-config="{{ ez_udw_config('multiple', {}) }}" class="btn btn-secondary ez-btn--select-subtree"{% if not form.limitation_type.location.vars.checked %} disabled{% endif %}>
+                            <button
+                                id="role_assignment_create_locations_select_content"
+                                data-input-selector="#{{ form.locations.vars.id }}"
+                                data-udw-config="{{ ez_udw_config('multiple', {}) }}"
+                                class="btn btn-secondary ez-btn--select-subtree"{% if not form.limitation_type.location.vars.checked %} disabled{% endif %}
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/user/role_assignment/create.html.twig:118]: This if is outside of the classes, shouldn't be in the next line?

**Review by dew326 — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by tischsoic — APPROVED** [review]

**Review by kacper-wieczorek-ibexa — APPROVED** [review]
QA - Approved. Tested on v3.2.7.

**Comment by dew326** [comment]
@lucasOsti can you merge it up?

**Comment by lucasOsti** [comment]
> @lucasOsti can you merge it up?

done

**Pull request merged**